app/config-schema.js

```diff
@@ -322,5 +322,23 @@ module.exports = {
         default: true
       }
     }
+  },
+  whitelabel: {
+    availableInFrontend: true,
+    bgImage: {
```

Comment:
Can we change this to `backgroundImage`?

Reply:
Of course